pertpy/tools/_mixscape.py

```diff
@@ -524,7 +524,7 @@ def plot_barplot(  # pragma: no cover
                   Infer the filetype if ending on {'.pdf', '.png', '.svg'}.
 
         Returns:
-            If show is False, return ggplot object used to draw the plot.
+            If `show==False`, return a :class:`~matplotlib.axes.Axes.
```

Comment:
Hmmmmm not sure how I feel about this one since this is not supposed to be actual code but rather an explanation.^^ Probably both are equally good and bad

Reply:
I think my change should be fine since many other docstring have also things like: "If `copy=True`, returns the copy of `adata` with the LDA result in `.uns`."